projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/subscription-billing/subscrption.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+export const serviceUser = {
```

Comment:
MAJOR: there is already a helper - projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/subscription-billing/subscription-billing.ts
Please try to re-use functions from here or add new functions to this file and use in your test